video-player-in-flutter-using-chewie-and-getx/index.md

```diff
@@ -0,0 +1,287 @@
+---
+layout: engineering-education
+status: publish
+published: true
+url: /engineering-education/video-player-in-flutter-using-chewie-and-getx/
+title: Video player in flutter using Chewie and Getx 
+description: This tutorial will guide you on how to set up and integrate Chewie and Getx in your Flutter application.
+author: kanoi-nikita
+date: 2021-11-05T00:00:00-11:00
+topics: [API]
+excerpt_separator: <!--more-->
+images:
+  - url: /engineering-education/video-player-in-flutter-using-chewie-and-getx/hero.png
+    alt: Video player in flutter using Chewie and Getx 
+--- 
+
+### Video player in Flutter using Chewie and Getx
+
+### Introduction
+A frequent requirement when building applications is playing video files. Doing so with Flutter is fairly straight forward, but there are a few things you should know about it. The code samples in this tutorial use Chewie for networking and Getx.
```

Comment:
Change to " A frequent requirement when building applications is playing video files. Doing so with Flutter is fairly straightforward, but you should know a few things about it. The code samples in this tutorial use Chewie for networking and Getx."